default.config.yml

```diff
@@ -194,6 +194,7 @@ installed_extras:
   - drupalconsole
   # - elasticsearch
   # - java
+  # - drush
```

Comment:
We should probably default to the common case where drush is installed globally, right?